### ISSUE TITLE
Build failed with an exception.

### ISSUE THREAD

**Issue by LDubya** [evrcargo/imapsize#4]
Hi @vadimkim, not sure what's going on but can't build. Please help:

> $ ./gradlew jar
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Could not determine java version from '11.0.2'.

 Java version: (Today I downloaded and installed a fresh version of the latest JDK and also the JRE.)

> $ java --version
> java 11.0.2 2019-01-15 LTS
> Java(TM) SE Runtime Environment 18.9 (build 11.0.2+9-LTS)
> Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.2+9-LTS, mixed mode)
> 
>

**Comment by vadimkim** [comment]
Software was developed for 1.8 JRE, but should work for 11 as well. Try to update gradle.

**Comment by LDubya** [comment]
@vadimkim, thanks! I managed to get it to build by switching to JRE 1.8, however I have a concern. For a 125mb mailbox, it took about 30 minutes to back up. Would this be normal? Or maybe something else is wrong? Internet connection here is ~50MBps so theoretically it should be able to backup 125MB within a minute.. maybe something is wrong with my setup?

**Comment by vadimkim** [comment]
Synchronization speed rather depends from the speed of mail server than network bandwidth. I used this software to migrate local mailboxes to the cloud. Since local mail server was under our control we were able to remove restrictions that influence backup speed and as far as I remember maximum speed was around 500Mb/min (2Gb mailbox downloads approx 5 min).
For public providers that serve a lot of mailboxes at the same time such a speed is just not allowed. They  try to share the load equally between clients and such "greedy" behaviour like downloading entire mailbox by IMAP can be just throttled down. Also keep in mind that usually restore and backup speeds are different. Restore considered to be "ingress" for any mailbox and is usually less restricted and thus - faster. 
If you are using big cloud provider (like outlook) they have kind on "batch export" mode for such kind of activity. Try to investigate more about what options does your provider offer if speed is an issue.
